model/schema.json

```diff
@@ -72,29 +72,36 @@
         },
         "data": {
           "description": "The opaque payload.",
-          "type": "object",
```

Comment:
Our custom [go json schema generator](https://github.com/elastic/go-json-schema-generate/) has some strange behaviour

If `type: object` is found it will define a go struct with that name then use it as the inline type for the attribute, If `format: raw` is found it will define the inline type as `json.RawMessage` (what we want for these cases).
If both are used, it leads to an unused (empty) struct definition.

Also names/attributes can collide, so if we have a top level object named `agent`, and an attribute named `agent` (defined with `type: object` and `format:raw`), things go _**wrong**_.

I removed `type: object` from attributes we want as a `json.RawMessage` in order to avoid this behaviour in the future